src/mintpy/cli/los2slope.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+#########################################################################
+# Program is part of MintPy                                             #
+# Copyright (c) 2013, Zhang Yunjun, Heresh Fattahi                      #
+# Author: Author: Ashok Dahal, Alexander Handwerger, Eric Fielding.     #
+# Mar 2024 Based on previous code by Antonio Valentino, Zhang Yunjun.   #
+#########################################################################
+
+import sys
+
+from mintpy.utils.arg_utils import create_argument_parser
+
+################################################################################
+REFERENCE = """reference:
+  Liu, L., Millar, C. I., Westfall, R. D., and Zebker, H. A.(2013). Surface motion of active
+    rock glaciers in the Sierra Nevada, California, USA: inventory and a case study using InSAR.
+    The Cryosphere, 7, 1109–1119.
+    https://doi.org/10.5194/tc-7-1109-2013
+
+  Handwerger, A.L., Huang, MH., Fielding, E.J. et al (2019). A shift from drought to extreme rainfall
+    drives a stable landslide to catastrophic failure. Sci Rep 9, 1569.
+    https://doi.org/10.1038/s41598-018-38300-0
+
+"""
+
+EXAMPLE = """example:
+ # Geocode the data before running this step
+ cd ./mintpy
+ mask.py velocity.h5 -m maskTempCoh.h5
+ geocode.py velocity_msk.h5 -l inputs/geometryRadar.h5 -x 0.00027778 -y -0.00027778 --bbox 32.0 32.5 130.1 130.5
+
+ # To run this program do the following
+ los2slope.py -file ./geo_velocity.h5 -g ./geo_geometryRadar.h5 -o ./geo_velocity_slp.h5 -m 15 --slp_thresh 1.0 --scaling_factor_thresh 10
```

Comment:
Its a bit unclear how to use the thresholds for example here -m 15 --slp_thresh 1.0 --scaling_factor_thresh 10 are suggested but the defaults are "-m 25 --slp_thresh 5.0 --scaling_factor_thresh 10".